### DISCUSSION TITLE
Resize dictionary transform with anti_aliasing?

### DISCUSSION THREAD

**Discussion by lidialuq** [Project-MONAI/MONAI#5215] [General]
I was using monai.transforms.Resized() with the argument anti_aliasing=True. I now need to switch to the dictionary version of this transform, Resized, and the argument is no longer accepted. What's the reason for this difference?

**Comment by wyli** [comment]
thanks, that's just not implemented yet... please see the progress tracker https://github.com/Project-MONAI/MONAI/issues/4264 cc @Nic-Ma

**Reply by wyli** [reply]
this is now added by @yiheng-wang-nv https://github.com/Project-MONAI/MONAI/pull/5223 thanks!
